Router/index.js

```diff
@@ -21,6 +21,12 @@ const request = require('request');
 const proxyTable = [
   { action: 'all', route: '/users/:userid/runs', target: 'http://runsservice:80' },
   { action: 'get', route: '/runs/:runid', target: 'http://runsservice:80' },
+  { action: 'post', route: '/adduser', target: 'http://usersservice:5000' },
```

Comment:
/adduser and /addfriend does not adhere to the RESTful paradigm, the HTTP verb should describe the action, the endpoint should describe the resource location /relationship